javascript/javascript1/week3/activity.js

```diff
@@ -0,0 +1,69 @@
+let activities = [];
+let date = new Date();
+function addActivity(date, activity, duration) {
+  activities.push({ date, activity, duration });
+}
+formattedDate =
+  date.getDate() + "-" + (date.getMonth() + 1) + "-" + date.getFullYear();
+addActivity(formattedDate, "Youtube", 30);
+addActivity(formattedDate, "facebook", 15);
+addActivity(formattedDate, "twetter", 10);
+console.log(activities);
+
+function showStatus(activities) {
+  let usageAmount = activities.reduce(function (r, a) {
+    return r + a["duration"];
+  }, 0);
+  for (i = 0; i < activities.length; i++) {
+    if (activities[i]["date"] == formattedDate) {
+      if (activities.length == 0) {
+        console.log("Add some activities before calling showStatus");
```

Comment:
You can check activities length at the beginning of the function and return if the condition is true like this
```js
  function showStatus(activities) {
    if (activities.length === 0) {
      console.log('Add some activities before calling showStatus');
      return;
    }
```